docs/runtimes/images.md

```diff
@@ -20,11 +20,11 @@ See [README](https://github.com/mlrun/mlrun/blob/development/dockerfiles/README.
 
 There is no difference in the usage between the MLRun images and external docker images. However:
 - MLRun images resolve auto tags: If you specify ```image="mlrun/mlrun"``` the API fills in the tag by the client version, e.g. changes it to `mlrun/mlrun:1.0.6`. So, if the client gets upgraded you'll automatically get a new image tag. 
-- Where the data node registry exists, MLRun: Appends the registry prefix, so the image loads from the datanode registry. This pulls the image more quickly, and also supports air-gapped sites. When you specify an MLRun image, for example `mlrun/mlrun:1.0.6`, the actual image used is similar to `datanode-registry.iguazio-platform.app.vm/mlrun/mlrun:1.0.6`.
+- Where the data node registry exists, MLRun Appends the registry prefix, so the image loads from the datanode registry. This pulls the image more quickly, and also supports air-gapped sites. When you specify an MLRun image, for example `mlrun/mlrun:1.0.6`, the actual image used is similar to `datanode-registry.iguazio-platform.app.vm/mlrun/mlrun:1.0.6`.
```

Comment:
Where the data node registry exists, MLRun appends the registry prefix, so the image loads from the datanode registry. This pulls the image more quickly, and also supports air-gapped sites. When you specify an MLRun image, for example `mlrun/mlrun:1.1.0`, the actual image used is similar to `datanode-registry.iguazio-platform.app.vm/mlrun/mlrun:1.1.0`.